src/language/records.md

```diff
@@ -31,20 +31,40 @@ type and as parameter types:
 }
 ```
 
-Named fields inside _record type annotations_ go inside a brace-delimited section
-of type and name pairs, after all positional fields:
+Named fields inside _record type annotations_ mirror how [named parameters and arguments][]
+work in functions. They go inside a curly brace-delimited
+section of type-and-name pairs, after all positional fields:
 
 ```dart
-// Record type annotation in a variable declaration
+// Record type annotation in a variable declaration:
 (String, String?, {int a, bool b}) record;
 ```
 
 Naming fields in record type annotations is optional for documentation purposes.
```

Comment:
**nit:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
Record type annotations don't require naming fields for documentation purposes.
```